packages/shared/src/components/plus/PlusInfo.tsx

```diff
@@ -85,90 +92,121 @@ const RadioGroupSkeleton = () => (
     ))}
   </div>
 );
+const getCopy = ({ giftToUser, title, description, subtitle }) => {
+  return {
+    titleCopy:
+      title ||
+      (giftToUser ? copy[PlusType.Gift].title : copy[PlusType.Self].title),
+    descriptionCopy:
+      description ||
+      (giftToUser
+        ? copy[PlusType.Gift].description
+        : copy[PlusType.Self].description),
+    subtitleCopy:
+      subtitle ||
+      (giftToUser
+        ? copy[PlusType.Gift].subtitle
+        : copy[PlusType.Self].subtitle),
+  };
+};
```

Comment:
Not tested, but could this be cleaner?
```suggestion
const getCopy = ({ giftToUser, title, description, subtitle }) => {
  const fallback = copy[giftToUser ? PlusType.Gift : PlusType.Self];
  return {
    titleCopy: title || fallback.title,
    descriptionCopy: description || fallback.description,
    subtitleCopy: subtitle || fallback.subtitle,
  };
};
```